### PR TITLE
HYDRA-777 : Clean test files and separate interactive and non interactive tests

### PR DESCRIPTION
**Pull request by lanierd-adsk**
Though it's not possible to use mayapy to run hydra tests (non interactively).

### CODE REVIEW

**Comment by lanierd-adsk** [comment]
FYI : @lilike-adsk @ppt-adsk @roopavr-adsk @vlasovi

File: test/lib/CMakeLists.txt [hunk 1]
```diff
@@ -1,55 +1,3 @@
-if(BUILD_MAYAUSD_LIBRARY)
```
Comment (lanierd-adsk) [inline test/lib/CMakeLists.txt:1]: These files don't exist in our repo.

File: test/lib/mayaUsd/CMakeLists.txt [hunk 2]
```diff
@@ -1,4 +1 @@
-add_subdirectory(fileio)
```
Comment (lanierd-adsk) [inline test/lib/mayaUsd/CMakeLists.txt:1]: These folders don't exist in our repo.

File: test/lib/mayaUsd/render/mayaToHydra/CMakeLists.txt [hunk 3]
```diff
@@ -1,5 +1,5 @@
-# Unit test scripts.
-set(TEST_SCRIPT_FILES
```
Comment (lanierd-adsk) [inline test/lib/mayaUsd/render/mayaToHydra/CMakeLists.txt:2]: The change is mainly add a non interactive section which is likely not to be usable with maya hydra tests. They all require a GUI (to use Qt) or need a viewport to set HdStorm.

File: test/lib/mayaUsd/render/mayaToHydra/CMakeLists.txt [hunk 4]
```diff
@@ -95,8 +99,45 @@ foreach(script ${TEST_SCRIPT_FILES_MESH_ADAPTER})
     )
 
     # Assign a CTest label to these tests for easy filtering.
-    set_property(TEST ${target} APPEND PROPERTY LABELS mayaHydra)
+    set_property(TEST ${target} APPEND PROPERTY LABELS mayaHydraMeshAdapter)
 endforeach()
 
+# Use Maya.exe with UI, interactive tests.
+    foreach(script ${INTERACTIVE_TEST_SCRIPT_FILES})
```
Comment (debloip-adsk) [inline test/lib/mayaUsd/render/mayaToHydra/CMakeLists.txt]: Minor, but this whole foreach block seems indented even though it isn't nested

**Review by debloip-adsk — DISMISSED** [review]

File: test/lib/mayaUsd/render/mayaToHydra/CMakeLists.txt (hunk 4 above)
Comment (lanierd-adsk) [reply test/lib/mayaUsd/render/mayaToHydra/CMakeLists.txt]: Thanks, I am updating the file.

**Review by debloip-adsk — APPROVED** [review]

**Review by ppt-adsk — APPROVED** [review]

**Pull request merged**